components/split-view/split-view.js

```diff
@@ -49,11 +48,12 @@ class SplitView extends LitElement {
 
   update(changedProperties) {
     super.update(changedProperties);
+    // this needs to handle children attributes changing
+    // maybe an event
     if (changedProperties.has('numberOfChildren')) this.handleResize();
```

Comment:
The change won't happen here, but the issue is as follows:
When you change the attribute of a child, the child count does not change. If multiple panes are being animated and hidden, the child count will not be updated, so width changes will not be reflected.